### PR TITLE
updated SAR benchmark

### PR DESCRIPTION
**Pull request by miguelgfierro**
### Description

Update SAR benchmark

### Checklist:

- [ ] My code follows the code style of this project, as detailed in our [contribution guidelines](../CONTRIBUTING.md).
- [ ] I have added tests.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: benchmarks/movielens.ipynb [hunk 1]
```diff
@@ -402,14 +402,14 @@
   },
```
[the remark this answers is not in the document]
Comment (miguelgfierro) [reply benchmarks/movielens.ipynb]: do you mean the prints that are commented?

[the remark this answers is not in the document]
Comment (miguelgfierro) [reply benchmarks/movielens.ipynb]: I removed them

**Comment by miguelgfierro** [comment]
In another PR, we need to fix https://github.com/Microsoft/Recommenders/issues/768, which involves several functions in the benchmarks and in the algos
cc @anargyri

**Comment by gramhagen** [comment]
what was the bug with using the enum in sar? the only downside i see with the current approach is SIMILARITY_TYPE[0] is not immediately obvious what it is.

**Comment by miguelgfierro** [comment]
`SIMILARITY_TYPE[similarity_type]` was not returning "jaccard", it returned an object. 

> the only downside i see with the current approach is SIMILARITY_TYPE[0] is not immediately obvious what it is.

Another option is to have a constant for each type, or directly add the string

**Comment by gramhagen** [comment]
yeah, i was taking the approach that we should just treat it as an enum not a string, you could access the string by similarity_type.jaccard.name if you need to, though using the enum objects would be better.

the reason behind that is it strictly enforces the types being used everywhere

**Comment by miguelgfierro** [comment]
> yeah, i was taking the approach that we should just treat it as an enum not a string, you could access the string by similarity_type.jaccard.name if you need to, though using the enum objects would be better. 
the reason behind that is it strictly enforces the types being used everywhere

I would argue that it's easier for the user if we use just a string for `self.similarity_type = similarity_type`. If you think having `SIMILARITY_TYPE[0]` is confusing, maybe we should have a variable, this is how it is done in Lightgbm for example https://github.com/microsoft/LightGBM/blob/master/python-package/lightgbm/basic.py#L174

**Comment by gramhagen** [comment]
should be transparent to the user as it will need to be a string in the constructor and processed accordingly there. but I'm fine with defining fixed strings
```
JACCARD = "jaccard"
COOCCUR = "cooccur"
...

... in init ...
if similarity_type not in [JACCARD, COOCCUR, ...]:
   raise
self.similarity_type = similarity_type

.... later ...
if self.similarity_type == JACCARD:
...

is that what you're thinking? if so I think it's clear and easy.

**Pull request merged**